.github/workflows/build.yml

```diff
@@ -84,6 +84,9 @@ jobs:
           nohup mvn quarkus:dev > app.log 2>&1 &
           PID=$(echo $!)
 
+          # Wait until the app installed the jokerequest custom resource definition
+          $SCRIPTS/waitFor.sh crd $K8S_NAMESPACE NAME jokerequests.samples.javaoperatorsdk.io
```

Comment:
what's `NAME` here?

Reply:
it's the column of the kubectl get crd command

Reply:
why is that needed?

Reply:
if the crd is not installed yet, it will print "resource does not exist". When it does exist, it will print the columns.

Reply:
OK, I think I got it, it's a hack to make the script work… can you add a comment for explaining why it's done this way, please?

Reply:
This is not a hack, but a text expectation. Having said that, in this case, it was not necessary. I've already updated the command.